app/src/main/res/layout/ticket_create_form.xml

```diff
@@ -255,5 +255,30 @@
 
             </android.support.design.widget.TextInputLayout>
         </LinearLayout>
+
+        <android.support.design.widget.TextInputLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
+
+            <EditText
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:inputType="number"
+                android:hint="@string/minOrder"
+                android:text="@={ BindingAdapters.integerToStr(ticket.minOrder) }" />
```

Comment:
As stated previously, this is not required. Please look in already created code how to implement input

Reply:
@iamareebjamal if i don't use integertoStr function then i get ' ' 'Error:(26, 56) error: cannot find symbol class TicketCreateFormBinding ' ' '

Reply:
What are you writing instead of this?

Reply:
@iamareebjamal  ` ` ` android:text="@={ ticket.minOrder }" ` ` ` if i don't use integerToStr

Reply:
Please post the log of build

Reply:
@iamareebjamal 
` ` ` /home/akshay/Documents/orga/open-event-orga-app/app/src/main/java/org/fossasia/openevent/app/common/data/models/Attendee.java
Warning:(124, 30) @Builder will ignore the initializing expression entirely. If you want the initializing expression to serve as default, add @Builder.Default. If it is not supposed to be settable during building, make the field final.
/home/akshay/Documents/orga/open-event-orga-app/app/src/main/java/org/fossasia/openevent/app/common/data/models/Event.java
Warning:(116, 29) @Builder will ignore the initializing expression entirely. If you want the initializing expression to serve as default, add @Builder.Default. If it is not supposed to be settable during building, make the field final.
Warning:(119, 29) @Builder will ignore the initializing expression entirely. If you want the initializing expression to serve as default, add @Builder.Default. If it is not supposed to be settable during building, make the field final.
/home/akshay/Documents/orga/open-event-orga-app/app/src/main/java/org/fossasia/openevent/app/common/data/models/Ticket.java
Warning:(62, 29) @Builder will ignore the initializing expression entirely. If you want the initializing expression to serve as default, add @Builder.Default. If it is not supposed to be settable during building, make the field final.
Warning:(65, 29) @Builder will ignore the initializing expression entirely. If you want the initializing expression to serve as default, add @Builder.Default. If it is not supposed to be settable during building, make the field final.
/home/akshay/Documents/orga/open-event-orga-app/app/build/generated/source/apt/debug/org/fossasia/openevent/app/databinding/TicketCreateLayoutBinding.java
Error:(26, 56) error: cannot find symbol class TicketCreateFormBinding
Error:(125, 72) error: cannot find symbol class TicketCreateFormBinding
Warning:File for type 'com.raizlabs.android.dbflow.config.OrgaDatabaseOrgaDatabase_Database' created in the last round will not be subject to annotation processing.
/home/akshay/Documents/orga/open-event-orga-app/app/src/main/java/org/fossasia/openevent/app/common/app/binding/DateBindings.java
Warning:(54, 24) Use of ObservableField and primitive cousins directly as method parameters is deprecated and support will be removed soon. Use the contents as parameters instead in method public static void bindDate(android.widget.Button,android.databinding.ObservableField<java.lang.String>)
Warning:(66, 24) Use of ObservableField and primitive cousins directly as method parameters is deprecated and support will be removed soon. Use the contents as parameters instead in method public static void bindTime(android.widget.Button,android.databinding.ObservableField<java.lang.String>)
Warning:View field eventStatistics collides with a variable or import
Warning:Use of ObservableField and primitive cousins directly as method parameters is deprecated and support will be removed soon. Use the contents as parameters instead in method org.fossasia.openevent.app.common.app.binding.DateBindings.bindDate
Warning:Use of ObservableField and primitive cousins directly as method parameters is deprecated and support will be removed soon. Use the contents as parameters instead in method org.fossasia.openevent.app.common.app.binding.DateBindings.bindTime
Warning:Use of ObservableField and primitive cousins directly as method parameters is deprecated and support will be removed soon. Use the contents as parameters instead in method org.fossasia.openevent.app.common.app.binding.DateBindings.bindDate
Warning:Use of ObservableField and primitive cousins directly as method parameters is deprecated and support will be removed soon. Use the contents as parameters instead in method org.fossasia.openevent.app.common.app.binding.DateBindings.bindTime
/home/akshay/Documents/orga/open-event-orga-app/app/src/main/res/layout/ticket_create_form.xml
Error:(275, 13) Cannot find the getter for attribute 'android:text' with value type java.lang.Integer on android.widget.EditText. 
Error:Execution failed for task ':app:compileDebugJavaWithJavac'.
> android.databinding.tool.util.LoggedErrorException: Found data binding errors. ` ` `

Reply:
https://stackoverflow.com/a/42260779

Reply:
@iamareebjamal I went through the following question before also, the problem is minOrder and maxOrder variables are not primitive integer variables but are variables of class Integer Wrapper class https://github.com/fossasia/open-event-orga-app/blob/a43ee671f9be9f7fe8f79a308393d79b2287a60d/app/src/main/java/org/fossasia/openevent/app/common/data/models/Ticket.java so the given solution doesn't work in this case, so i had to use the same method used for all other wrapper classes such as Float, Double etc.

Reply:
Alright, please resolve the suggestion about PMD

Reply:
Alright, please resolve the suggestion about PMD

Reply:
@iamareebjamal i looked alot for the specific supresswarnings statement but i dind't find any for the issue ` ` ` Assigning object to null is code smell, consider refractoring ` ` `

Reply:
Google Search First Page, 3rd result (PMD Website), 2nd rule